### ISSUE TITLE
Add ability to specify Path Routes in Listener

### ISSUE THREAD

**Issue by hyder** [oracle/terraform-provider-oci#487]
Terraform Version
v0.11.2

OCI Provider Version
v2.0.6

Description:
This is related to #486 and dependent on #434 . The OCI console allows specifying a path route in the listener.

Terraform Plan
This is an enhancement request

**Comment by alexng-canuck** [comment]
Thanks for the request. This is an upcoming feature in our queue.

You can check #434 for updates on when this feature becomes available. Let us know if you think this request isn't covered in #434.

**Comment by alexng-canuck** [comment]
Closing as duplicate of #434. Please reopen this if we misunderstood the request and #434 doesn't address this.

**Comment by anandhmahal** [comment]
Support for path routes was added in release 2.1.2
